src/patterns/dosdp-patterns/cellPartOfAnatomicalEntity.yaml

```diff
@@ -2,25 +2,30 @@ pattern_name: cellPartOfAnatomicalEntity
 pattern_iri: http://purl.obolibrary.org/obo/cl/cellPartOfAnatomicalEntity
 
 description: A cell that is part of an anatomical entity.
-#examples: CL_0009001 compound eye retinal cell, CL_2000004 pituitary gland cell (654 total)
+# examples:
+# CL_0009001 compound eye retinal cell,
+# CL_2000004 pituitary gland cell (654 total)
```

Comment:
We should be able to manage to some better examples.  I don't like these 'any cell type' X location terms.

"Peyer's patch B cell"   is better than the pituitary gland example.  The compound eye example is really just a way of burying explicit taxon declaration, which also makes it a slightly odd example.